src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-100000.mdx

```diff
@@ -0,0 +1,98 @@
+
+## Deprecations
+
+* Remove Python 2.7 Support
+
+  * Removes compatibility with Python 2.7. Also removes support for the following libraries that were only supported on Python2.7:
+    * [umemcache](https://pypi.org/project/umemcache/)
+    * [oursql](https://pypi.org/project/oursql/)
+    * [pywapi](https://pypi.org/project/pywapi/)
+    * [Pylons](https://pypi.org/project/Pylons/)
+    * [web2py](https://pypi.org/project/web2py/)
+    * [weberror](https://pypi.org/project/WebError/)
+
+## New features
+
+* Add Kafka server metric
+
+  * Adds a metric to capture the host and port information for Kafka entities on both consumers and producers. This metric allows the UI to link AWS information with AWS MSK entities in the service map.
+
+* Add capturing of memcache `host` and `port`
+
+  * Adds capturing of `host` and `port` info on the following memcache libraries: [bmemcached](https://pypi.org/project/python-binary-memcached/), [aiomcache](https://pypi.org/project/aiomcache/), and [pymemcache](https://pypi.org/project/pymemcache/). This allows the UI to link AWS information with AWS memcache entities in the service map.
+
+* Add support for [kafka-python-ng](https://pypi.org/project/kafka-python-ng/)
+
+  * [kafka-python](https://pypi.org/project/kafka-python/) has been released under a new name: [kafka-python-ng](https://pypi.org/project/kafka-python-ng/). This continues to support kafka-python under the new package name.
+
+* Add support for [uvicorn_worker](https://pypi.org/project/uvicorn-worker/)
+
+  * `uvicorn.workers` has been moved to a separate package called `uvicorn_worker`. This checks for both names of the module when reporting dispatcher information.
+
+* Automatically detect function signature in [LangChain](https://pypi.org/project/langchain/) `similarity_search`
+
+  * Automatically detect the function signature when wrapping `similarity_search`.
+
+* Add environment variable for garbage collector runtime metrics
+
+  * Garbage collector runtime metircs can now be enabled/disabled via the following environment variable: `NEW_RELIC_GC_RUNTIME_METRICS_ENABLED`. 
+
+## Bug fixes
+
+* Fix bug in gRPC entity name detection
+
+  * Previously, the gRPC channel entity name was missing the first character(s) in specific cases. This bug has been fixed.
+
+* Fix issue in [LangChain](https://pypi.org/project/langchain/) where a thread is started and the transaction context is lost
+
+  * Previously, when calling some chains like retreival chains, the transaction would be lost and instrumentation would be broken because a thread would be started by [LangChain](https://pypi.org/project/langchain/). This has been fixed so the transaction context is passed across threads and instrumentation still works for retreival chains.
```

Comment:
```suggestion
  * Previously, when calling some chains like retrieval chains, the transaction would be lost and instrumentation would be broken because a thread would be started by [LangChain](https://pypi.org/project/langchain/). This has been fixed so the transaction context is passed across threads and instrumentation still works for retrieval chains.
```